services/m3dbnode/config/config_test.go

```diff
@@ -186,6 +186,16 @@ pooling:
         capacity: 4096
         lowWatermark: 0.01
         highWatermark: 0.02
+    tagArrayPool:
+        size: 4096
+        capacity: 8
+        maxCapacity: 32
+        lowWatermark: 0.01
+        highWatermark: 0.02
+    indexResultsPool:
+        size: 4096
```

Comment:
Probably won't be more than 4k distinct queries at any one point in time I guess, 8192 might be safer though.

Reply:
Sure thing

Reply:
done